### ISSUE TITLE
Wrong gene coordinate for UBE2V2

### ISSUE THREAD

**Issue by pwwang** [biothings/mygene.info#48]
http://mygene.info/v3/gene/7336
```json
"genomic_pos_hg19": [
    {
      "chr": "20",
      "end": 48732496,
      "start": 48697661,
      "strand": -1
    },
    {
      "chr": "8",
      "end": 48977268,
      "start": 48920960,
      "strand": 1
    }
  ]
```
The right coordinate should be the second one. No sure where is the first one from.

**Comment by sirloon** [comment]
1st column: gene ID
2nd column ( with "|" as separator): chromosomes found in genomic_pos_hg19 (only when it's a list)
3rd column: chromosomes found in genomic_pos (hg38) if any

[poshg19hg38.txt](https://github.com/biothings/mygene.info/files/2291377/poshg19hg38.txt)

**Comment by newgene** [comment]
After looking at the list @sirloon posted above, we can filter down to a list of 714 rows, which could be "fixed" based on their hg38 pos (when hg38 position contains position from a single "chr" value, and that chr appears only once in hg19 position):

[poshg19hg38_fixable_by_hg38.txt](https://github.com/biothings/mygene.info/files/2291659/poshg19hg38_fixable_by_hg38.txt)

However, I would like to hold off this fix for now. Ideally, we should get those hg19 genomic pos data from NCBI, then this issue will be fixed from the data source.

I'm closing this issue for now, with the reference to this new issue I just created #50.
